spec/index.bs

```diff
@@ -15831,6 +15838,24 @@ This feature adds the following [=optional API surfaces=]:
     - {{GPUTextureFormat/"bc7-rgba-unorm"}}
     - {{GPUTextureFormat/"bc7-rgba-unorm-srgb"}}
 
+<h3 id=texture-compression-bc-sliced-3d data-dfn-type=enum-value data-dfn-for=GPUFeatureName>`"texture-compression-bc-sliced-3d"`
+<span id=dom-gpufeaturename-texture-compression-bc-sliced-3d></span>
+</h3>
+
+Allows the {{GPUTextureDimension/3d}} dimension for textures with BC compressed formats.
+
+Note:
+This feature does not enable the BC formats, so in order to take advantage of it,
+enable the {{GPUFeatureName/"texture-compression-bc"}} feature as well.
+
+Note: Adapters which support either {{GPUFeatureName/"texture-compression-bc"}}
+or {{GPUFeatureName/"texture-compression-bc-sliced-3d"}}
+always support both, even though they are separate features.
+To use {{GPUFeatureName/"texture-compression-bc-sliced-3d"}},
+both must be enabled explicitly.
```

Comment:
these notes are a bit redundant, I think they can be edited down into one.

Reply:
@kainino0x agreed, I tried to combine by adding " as this feature does not enable the BC formats." to second note and removing the first one. PLMK if that's positive otherwise we can change